server/routes/nextPayment.routes.js

```diff
@@ -84,4 +84,27 @@ router.get(
   NextPaymentController.recalculateNextPayment,
 );
 
+/**
+ * @swagger
+ * /next-payment/cronjob:
```

Comment:
The essence of the endpoint is to resolve expired next payments. 
```suggestion
 * /next-payment/resolved-expired:
```